app/views/bars/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1><%= @bar.name %></h1>
+
+<p><%= @bar.address %></p>
+
+<p><%= @bar.city %></p>
+
+<p><%= @bar.state %></p>
+
+<p><%= @bar.zip %></p>
+
+<p><%= @bar.description %></p>
+
+<p><%= link_to "#{@bar.url}", @bar.url %></p>
+
+<p><%= image_tag ("#{@bar.photo_url}") %></p>
+
+<p><%= @bar.seating_capacity %></p>
+
+<p><%= @bar.food %></p>
+
+<p><%= @bar.outdoor_seating %></p>
+
+<p><%= @bar.pet_friendly %></p>
+
+<p><%= link_to "Edit #{@bar.name}", edit_bar_path(@bar.id) %></p>
+<% if user_signed_in? %>
```

Comment:
`<% if user_signed_in? && current_user.admin? %>` can combine logic of line 26 and 27 together.